docs/documentation/bundles/utilities.rst

```diff
@@ -14,24 +18,46 @@ Core metadata generation, testing and citation formatting.
     - Python library for generating Dublin Core XML from Python dictionaries.
 - `dictdiffer <https://dictdiffer.readthedocs.io>`_
     - Python library for diffing/patching/merging JSON documents.
+- `docker-services-cli <https://docker-services-cli.readthedocs.io>`_
+    - Infrastructure services for local and CI tests.
 - `dojson <https://dojson.readthedocs.io>`_
     - JSON to JSON rule-based transformation library.
+- `domapping <https://domapping.readthedocs.io/en/latest/>`_
+    - Package generating elasticsearch mapping from jsonschemas.
+- `doschema <https://doschema.readthedocs.io/en/latest/>`_
+    - Utilities to work with JSON Schemas.
 - `flask-breadcrumbs <https://flask-breadcrumbs.readthedocs.io>`_
     - Flask extension for managing breadcrumbs in web applications.
 - `flask-celeryext <https://flask-celeryext.readthedocs.io>`_
     - Celery integration for Flask.
+- `flask-cli <https://flask-cli.readthedocs.io>`_
+    - Flask-CLI is a backport of Flask 1.0's new click integration to Flask.
 - `flask-iiif <https://flask-iiif.readthedocs.io>`_
     - IIIF server for Flask.
+- `flask-kvsessions <https://pythonhosted.org/Flask-KVSession/>`_
+    - A drop-in replacement for Flask's session handling using server-side sessions.
 - `flask-menu <https://flask-menu.readthedocs.io>`_
     - Menu generation support for Flask.
+- `flask-resources <https://flask-resources.readthedocs.io>`_
+    - REST APIs for Flask
+- `flask-security-fork <https://flask-security-fork.readthedocs.io>`_
```

Comment:
This one should also be left (i've also archived it now.

Reply:
`flask-security-fork` removed